std/traits.d

```diff
@@ -1498,7 +1501,6 @@ if (func.length == 1 && isCallable!func)
                     // like this.
                     auto " ~ val ~ " = " ~ args ~ "[0];
                     auto " ~ ptr ~ " = &" ~ val ~ ";
-                        // workaround Bugzilla 16582
```

Comment:
Why was this removed?

Reply:
It's already mentioned 7 lines above (expand the diff upwards)

Reply:
Ah ok, I did expand but didn't see it. My brain is numbed by this review lol.